ludwig/models/llm.py

```diff
@@ -252,8 +317,25 @@ def initialize_adapter(self):
             self.model.print_trainable_parameters()
             logger.info("==================================================")
 
+    @property
+    def trained_using_adapter(self) -> bool:
+        return bool(self.config_obj.adapter)
```

Comment:
Is it possible for the config_obj.adapter to be non-None/non-empty when it's not set by the user explicitly?

Reply:
I think by default it is set to `None` if not specified. In other parts of the code, we check for if self.config_obj.adapter, and then load the adapter weights, so this is effectively equivalent to that.